src/client/src/apps/MainRoute/widgets/analytics/components/analyticsLineChart/AnalyticsLineChart.tsx

```diff
@@ -80,7 +80,7 @@ const LineCharts: React.FC<LineChartProps> = React.memo(props => {
   useEffect(() => {
     const newOffset = offsetState + 1 === intervalWidth ? 0 : offsetState + 1
     setOffsetState(newOffset)
-  }, [props])
+  }, [props, offsetState])
```

Comment:
@fdore do we need `props` here as a dependency? It doesn't look right to me.

Reply:
`offsetState` should be enough

Reply:
actually, I don't think it makes sense to have a dep on`offsetState` here, given the only place were we change it is inside the effect

Reply:
should I leave it as an empty array then?

Reply:
Ah, got it. It's dependent on the changes in `model.seriesData`.